src/layout/Breadcrumbs.tsx

```diff
@@ -25,7 +24,7 @@ const Breadcrumbs: React.FC = () => {
 
   return (
     <>
-      {pathNames.length > 0 && (
+      {pathNames.length && (
```

Comment:
Fant ut at dette var litt problematisk å bruke i en render-return. For hvis `pathNames.length` er 0 her, så vil denne blokka returnere `0`. Så vi kan visst ikke gjøre dette her :/

Reply:
hehhe, så det nå. skal fikse